specs/Deployment/MSIXPackageVersioning.md

```diff
@@ -520,3 +520,30 @@ length constraints. The specific packages Names in Windows App SDK 1.0:
 * WARddlm: `Microsoft.WinAppRuntime.DDLM.<major>.<minor>.<build>.<revision>-<shortarchitecture>[-shorttag]`
 
 See [2.5. Decision 5: Package Names](#25-decision-5-package-names) for more details.
+
+
+## Version Starting 2.0
+In 2.0, the family name of the MSIX package changes. The new style will be:
+* WARfwk: `Microsoft.WindowsAppRuntime.<rmajor>[-tag]`
+* WARmain: `MicrosoftCorporationII.WinAppRuntime.Main.<rmajor>[-shorttag]`
+* WARsingleton: `MicrosoftCorporationII.WinAppRuntime.Singleton[-shorttag]`
+* WARddlm: `Microsoft.WinAppRuntime.DDLM.<major>.<minor>.<Patch>.<BuildNumber>-<shortarchitecture>[-shorttag]`
+
+The version will also change:
+<Major>.<Minor>.<Patch>.<BuildNumber>
```

Comment:
><BuildNumber> [](http://example.com/codeflow?start=24&length=13)

Given "MSIX version's 4th field (Revision) is reserved for security updates", I think we could just list this as "0".